src/pivot-table/components/containers/ScrollableContainer.tsx

```diff
@@ -1,26 +1,40 @@
 import React from "react";
-import type { Rect } from "../../../types/types";
 import { useBaseContext } from "../../contexts/BaseProvider";
 
+export enum ScrollableContainerOrigin {
```

Comment:
Put this in `src/types/types.ts` since it's used in a lot places? Not only this file and its test file.